MethodDecorator.Fody.Tests/TestsBase.cs

```diff
@@ -16,8 +16,8 @@ public abstract class TestsBase : IDisposable {
 
         protected IList<Tuple<Method, object[]>> Records {
             get {
-                var records = (IList<Tuple<int, object[]>>)this.RecordHost.Records;
```

Comment:
To be honest, I am not sure if it is actually better with the preprocessor directives or not.